runtime/src/lib.rs

```diff
@@ -609,6 +610,48 @@ where
     }
 }
 
+impl DispatchAggregation for Runtime {
+    fn dispatch_aggregation(
+        domain_id: u32,
+        aggregation_id: u64,
+        aggregation: H256,
+        destination_params: Destination,
+    ) -> DispatchResult {
+        match destination_params {
+            Destination::None => Ok(()),
+            Destination::Hyperbridge(params) => {
+                pallet_hyperbridge_aggregations::Pallet::<Runtime>::dispatch_aggregation(
+                    pallet_hyperbridge_aggregations::PALLET_ID.into_account_truncating(),
+                    Params {
+                        domain_id,
+                        aggregation_id,
+                        aggregation,
+                        module: params.destination_module,
+                        destination: StateMachine::from(params.destination_chain),
+                        timeout: params.timeout,
+                        fee: Balance::zero(),
```

Comment:
We don't want to bind in any way this parameter here and the Delivery.price parameter in the aggregate pallet ?

Reply:
This parameter now doesn't have any sense and it's not clear how to use it in the feature. So we preferred to not speculate over it and left it 0.

Reply:
Please let's add a TODO to clarify